<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>EditorMiscWidget</class>
 <widget class="QWidget" name="EditorMiscWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>583</width>
    <height>528</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QCheckBox" name="chkReadonlySystemHeaders">
     <property name="text">
      <string>Open system header files in read only mode</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QCheckBox" name="chkLoadLastFiles">
     <property name="text">
      <string>Auto load files being open when Red Panda C++ last exited.</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QCheckBox" name="chkParseTodos">
     <property name="text">
      <string>Parse TODOs</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QWidget" name="widget" native="true">
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QWidget" name="widget_2" native="true">
     <layout class="QHBoxLayout" name="horizontalLayout_3">
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox_3">
     <property name="title">
      <string>Action before saving files</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_2">
      <item row="0" column="1">
       <widget class="QRadioButton" name="rbAutoReformat">
        <property name="text">
         <string>Reformat</string>
        </property>
       </widget>
      </item>
      <item row="0" column="2">
       <widget class="QRadioButton" name="rbRemoveTrailingSpaces">
        <property name="text">
         <string>Trim trailing spaces</string>
        </property>
       </widget>
      </item>
      <item row="0" column="0">
       <widget class="QRadioButton" name="rbNone">
        <property name="text">
         <string>None</string>
        </property>
       </widget>
      </item>
      <item row="0" column="3">
       <spacer name="horizontalSpacer_2">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QCheckBox" name="chkAutoDetectFileEncoding">
     <property name="text">
      <string>Auto detect encoding when openning files</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox_2">
     <property name="title">
      <string>Default file encoding</string>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QComboBox" name="cbEncoding"/>
      </item>
      <item>
       <widget class="QComboBox" name="cbEncodingDetail"/>
      </item>
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>Default file type</string>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_4">
      <item>
       <widget class="QRadioButton" name="rbCFile">
        <property name="text">
         <string>C files</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="rbCppFile">
        <property name="text">
         <string>C++ files</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer_3">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>117</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>chkReadonlySystemHeaders</tabstop>
  <tabstop>chkLoadLastFiles</tabstop>
  <tabstop>chkParseTodos</tabstop>
  <tabstop>rbNone</tabstop>
  <tabstop>rbAutoReformat</tabstop>
  <tabstop>rbRemoveTrailingSpaces</tabstop>
  <tabstop>chkAutoDetectFileEncoding</tabstop>
  <tabstop>cbEncoding</tabstop>
  <tabstop>cbEncodingDetail</tabstop>
  <tabstop>rbCFile</tabstop>
  <tabstop>rbCppFile</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
